NEWS

```diff
@@ -1,9 +1,117 @@
 python-scrapinghub release notes
 ================================
 
+Version 1.9.0
+-------------
+(release on Nov 2nd, 2016)
```

Comment:
that's strange @redapple we'll change the headers later if starting digit will stand in our way :)